packages/terra-form-select/CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## Unreleased
 
+* Removed
```

Comment:
Should we consider this breaking? I lean towards no.

Reply:
Looks like we won't break if a placeholder prop continues to be provided. Doesn't seem breaking to me.

Reply:
Done.